src/service/repo/anonUserInfoRepo.js

```diff
@@ -1,12 +1,12 @@
-import api from '../config';
+import anonymousApi from '../anonymousConfig';
 
 const baseUrl = {
   getPostAnonUserInfo: '/feeds/generate-anonymous-username'
 };
 
 const baseGetAnonUserInfo = async (contentType, postId = '') => {
   try {
-    const response = await api.post(baseUrl.getPostAnonUserInfo, {
+    const response = await anonymousApi.post(baseUrl.getPostAnonUserInfo, {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `api` to `anonymousApi` seems to be a significant one. If `anonymousApi` is specifically configured for anonymous user-related operations, this change makes sense. However, it's important to ensure that `anonymousApi` has all the necessary configurations and error handling mechanisms as the previous `api`. Also, make sure that the response from `anonymousApi.post` is handled properly in the calling function.
```diff
- import api from '../config';
+ import anonymousApi from '../anonymousConfig';

  const baseUrl = {
    getPostAnonUserInfo: '/feeds/generate-anonymous-username'
  };
  
  const baseGetAnonUserInfo = async (contentType, postId = '') => {
    try {
-     const response = await api.post(baseUrl.getPostAnonUserInfo, {
+     const response = await anonymousApi.post(baseUrl.getPostAnonUserInfo, {
        contentType,
        postId
      });
```
Please ensure that the switch to `anonymousApi` does not introduce any unexpected behavior or errors due to differences in configuration between `api` and `anonymousApi`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->